README.md

```diff
@@ -266,29 +269,29 @@ type             16 bytes     64 bytes    256 bytes   1024 bytes   8192 bytes
 aes-128-cbc     559001.23k   603807.10k   615512.66k   619206.31k   617106.09k
 ```
 
-A normal CPU can provide a 6x increment in performance just by setting a flag, nice!
+A normal CPU can provide a 6x increment in performance just by setting a flag. Nice!
 
 
 # Protocol Definition
-Time to talk on how this works with a blockchain. The scenario is the one described before: *Alice* just spent some bitcoins to buy a digital asset *X* from *Bob*.
+Time to talk about how this works with a blockchain. The scenario is the one described before: *Alice* just spent some bitcoin to buy a digital asset *X* from *Bob*.
 
 Ideally, Alice and Bob will pass the same transaction around, creating an audit trail of their handshake.
 
 
 ## Step 0: Bob publishes metadata about his payload
-Bob has few private assets he is willing to share under certain conditions. For each asset, he creates a transaction and push it to the network. Those transactions contain metadata about the assets. We haven't defined yet a schema for metadata, but they will be expressive enough to cover those (and more) use cases:
+Bob has few private assets he is willing to share under certain conditions. For each asset, he creates a transaction and push it to the network. Those transactions contain metadata about the assets. We haven't defined a schema for metadata yet, but the schema will be expressive enough to cover these use cases and more:
 
 - sell an asset for a fixed price
 - give read permission to an asset for a time interval
 - etc.
 
 If needed, each payload offer can have an extra field with the stake that Bob puts to vouch the offer. This stake must be read as "if I'm not delivering what I promise, the stake is yours".
 
-(A reputation system can be used, both as a complementary guarantee, or as an alternative to stake.)
+A reputation system could be used, both as a complementary guarantee, or as an alternative to stake.
 
 
 ## Step 1: Alice starts a negotiation with Bob.
-Alice transfers a transaction to Bob. Bob has to reply to **know** what Alice wants and do actual work. This is of course susceptible to spam, so Alice puts stake to prove she is not spamming him. If there is a trust relationship between Alice and Bob, she might omit the stake.
+Alice starts the handshake by transferring an asset to Bob. Bob has to reply to **know** what Alice wants and do actual work. This is of course susceptible to spam, so Alice puts stake to prove she is not spamming Bob. If there is a trust relationship between Alice and Bob, she might omit the stake.
```

Comment:
This change raises another technical question.

@vrde Does Alice's transfer of the asset to Bob start the handshake? That doesn't seem right to me.

Reply:
Yes, it's like a browser starting a handshake with a server.
In this case Bob will find in his _unspents_ an asset that signals the start of a handshake. Does it sound reasonable?

Reply:
ok